### PR TITLE
[stable/gitlab-ce] Https probes support

### LINKED ISSUE #1970: [stable/gitlab-ce] Support for https liveness/readyness probes
**Is this a BUG REPORT or FEATURE REQUEST?**:
This is a feature request.

**Version of Helm and Kubernetes**:
Any

**Which chart**:
stable/gitlab-ce

**What happened**:
When exposing gitlab-ce with https and using the http-to-https-redirect feature, the liveness and readiness probes will fail because the /help endpoint with http will respond a 302 redirect to https.

**What you expected to happen**:
The `deployment.yml` file has the `port` fields on the liveness and readiness probes hardcoded to `http`. Simply allowing these fields to be overridden in `values.yml` would solve this issue.

### PR DESCRIPTION
**Pull request by andreasevers**
Closes #1970

### CODE REVIEW

File: stable/gitlab-ce/templates/deployment.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ spec:
         livenessProbe:
           httpGet:
             path: /help
-            port: http
+            port: {{ default "http" .Values.livenessPort }}
```
Comment (prydonius) [inline stable/gitlab-ce/templates/deployment.yaml]: the default is not needed if it's set in `values.yaml`

**Comment by prydonius** [comment]
@andreasevers the chart (minor) version should also be bumped here

**Comment by andreasevers** [comment]
If someone could squash the commits on merge, that'd be awesome :)

**Comment by andreasevers** [comment]
@prydonius would this constitute a minor? Since the change is that small and not breaking backwards-compatibility I thought a patch would be sufficient. Either way, I'll change to whatever you prefer.

**Pull request merged**